integrations/pinterest-tag/lib/index.js

```diff
@@ -63,15 +63,27 @@ Pinterest.prototype.identify = function(identify) {
 
 Pinterest.prototype.page = function(page) {
   // If we have a category, the use ViewCategory. Otherwise, use a normal PageVisit.
+  var pinterestPageProps = {
+    name: page.name() || ''
+  };
+
+  var eventKeys = ['event_id', 'eid', 'eventID'];
```

Comment:
Why do we search for `eid`, `eventID` these specific attributes? How did we decide these three values?

Reply:
[Pinterest only allows event IDs in these three formats. ](https://help.pinterest.com/en/business/article/add-event-codes#:~:text=advertiser%20account%20currency.-,event_id,-eventId0001)

Reply:
Okay. So, do we expect customers to provide these values in properties? Is this consistent with how we can handle `track`?

Reply:
Yes, for `track` events, mappings are generated through the `generatePropertiesObject` function. Using forLoop, when the user adds any of these three event IDs, that value directly gets attached to the mappings.